go.mod

```diff
@@ -22,9 +22,10 @@ require (
 	github.com/jackc/pgservicefile v0.0.0-20221227161230-091c0ba34f0a // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/riverqueue/river/riverdriver/riverpgxv5 v0.0.0-20231110014757-1a7176abdf3c // indirect
```

Comment:
For now I had to lock this to the commit hash in this branch, which is obviously problematic, but it's a bit of a chicken and egg problem. I can't `go get` the new module without specifying this branch. We can fix if/when this comes in.

Reply:
Hmm, does `bun` [get away](https://github.com/uptrace/bun/blob/master/go.mod) with not needing this because they don't directly use their driver packages within the main package? I'm just wondering if this is going to cause more pain than it's worth right now, like will we have to bump this with every version and always run into a chicken & egg issue?

Reply:
I looked into that, and they get away with it by making every single example in the examples directory its own Go module as well :$ (You also can't use the specific driver in tests, which we do a lot.)

Maybe something we should do eventually, but man, kind of the nuclear option.

I figure that we could probably keep the dependency for now, and one day when the project's more mature, detach it by following a path closer to Bun's. (Removing the dependency would not be a breaking change, which is good.)

Reply:
So how do we actually publish an updated `riverdriver/riverpgxv5` and a plain `river` at the same time? Do we need to do a double tag and release?

Reply:
Yeah I think that's basically it ...

If you take a look at Bun, the latest release is v0.1.17 and its examples are pegged to v1.1.16.

It kind of makes sense. If the two modules lived in separate Git repos, you'd need to do a double-release there too.

Reply:
I have a feeling this is going to be painful and error prone but let's give it a shot, I don't think Go gives us any better options 😆